docs/book/src/developer/tilt.md

```diff
@@ -1 +1,86 @@
+
+## Overview
+
+This document describes how to use [kind](https://kind.sigs.k8s.io) and [Tilt](https://tilt.dev) for a simplified workflow that offers easy deployments and rapid iterative builds.
+
+## Prerequisites
+
+1. [Docker](https://docs.docker.com/install/) v19.03 or newer
+2. [kind](https://kind.sigs.k8s.io) v0.9 or newer (other clusters can be
+   used if `preload_images_for_kind` is set to false)
+3. [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+4. [Tilt](https://docs.tilt.dev/install.html) v0.22.2 or newer
+5. [envsubst](https://github.com/drone/envsubst) or similar to handle
+   clusterctl var replacement
+6. Clone the [Cluster API](https://github.com/kubernetes-sigs/cluster-api) repository
+   locally
+7. Clone the [cluster-api-provider-ibmcloud](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud) repository you want to deploy locally as well
+
+## Create a kind cluster
+
+First, make sure you have a kind cluster and that your `KUBECONFIG` is set up correctly:
+
+``` bash
+kind create cluster
+```
+
+This local cluster will be running all the cluster api controllers and become the management cluster which then can be used to spin up workload clusters on IBM Cloud.
+
+## Create a tilt-settings.json file
+
+Next, create a `tilt-settings.json` file and place it in your local copy of `cluster-api`. Here is an example:
+
+**Example `tilt-settings.json` for CAPI-IBM clusters:**
+
+```json
+{
+    "default_registry": "gcr.io/you-project-name-here",
+    "provider_repos": ["../cluster-api-provider-ibmcloud"],
+    "enable_providers": ["ibmcloud", "kubeadm-bootstrap", "kubeadm-control-plane"],
+    "kustomize_substitutions": {
+      "IBMCLOUD_API_KEY": "XXXXXXXXXXXXXXXXXX"
+    }
+}
+```
+**NOTE**: For information about all the fields that can be used in the `tilt-settings.json` file, check them [here](https://cluster-api.sigs.k8s.io/developer/tilt.html#tilt-settingsjson-fields).
+
+## Run Tilt
+
+To launch your development environment, run:
+
+``` bash
+tilt up
+```
+
+Kind cluster becomes a management cluster after this point, check the pods running on the kind cluster by running `kubectl get pods -A`.
+
+## Create workload clusters
+
+Set the following variables for CAPI-IBM managed clusters. Use clusterctl to render the yaml through templates and deploy the cluster:
+
+```bash
+IBMPOWERVS_SSHKEY_NAME="my-pub-key" \
+IBMPOWERVS_VIP="192.168.151.22" \
+IBMPOWERVS_VIP_EXTERNAL="158.175.162.22" \
+IBMPOWERVS_VIP_CIDR="29" \
+IBMPOWERVS_IMAGE_NAME="capibm-powervs-centos-8-1-22-4" \
+IBMPOWERVS_SERVICE_INSTANCE_ID="7845d372-d4e1-46b8-91fc-41051c984601" \
+IBMPOWERVS_NETWORK_NAME="capi-test-3" \
+clusterctl generate cluster ibm-powervs-1 --kubernetes-version v1.22.4 \
+--target-namespace default \
+--control-plane-machine-count=3 \
+--worker-machine-count=1 \
+--from ./cluster-template-powervs.yaml | kubectl apply -f -
+```
+Check the tilt logs and wait for the clusters to be created.
+
```

Comment:
lets not repeat the content here, we can refer to the already existing doc which talks about the workload cluster deployment for both vpc, powervs